modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -922,6 +922,135 @@ GAPI_FLUID_KERNEL(GFluidNot, cv::gapi::core::GNot, false)
     }
 };
 
+//--------------------------------------
+//
+// Fluid math kernels: bitwise with Scalar
+//
+//--------------------------------------
+
+static void convertScalarForBitwise(const cv::Scalar &_scalar, int scalarI[4])
```

Comment:
Also,

It could return an `std::array<int,4>` so then your caller's code would be more clear.

It is so C-ish right now.

Reply:
OK, reworked as you proposed, looks better for me, thanks!